apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Data insights on PWAs
+excerpt: A quick read on some insights derived from our  analytics on PWABuilder.com
+description: This blog presents a snapshot of data from Aug - Oct 2023 of all the PWAs that have been through the flow of PWABuilder.com and what we can learn from them.
+date: 2023-11-02
+updatedDate: 2023-11-02
+trending: true
+featured: true
+image: posts/data-insights-on-PWAs/titleimage.png
+isPost: true
+backUrl: '/'
+author:
+  name: Amrutha Srinivasan
+  title: PWABuilder Engineer
+tags:
+  - post
+  - PWA
+  - Insights
+---
+
+
+Every day, PWABuilder analyzes thousands of PWAs. We use this data to learn from you, the developers, about all the current trends in PWAs based on the kind of PWAs you create. In this article, we present some data and insights gathered from a 3-month snapshot of our analytics (Aug 2023 – Oct 2023). 
+During this period, we recorded a total of 10,640 unique URLs that went through the flow of PWABuilder. That’s a PWA data mine right there! 
+Manifests and service workers are the heart and soul of a PWA. They distinguish your PWA from being just another website. Manifests define how your PWA looks and behaves when installed on a device. Service workers are essentially JavaScript workers that run in the background and are used for features like notifications, offline support, background sync, and many more. They can also be used just to fetch data from APIs for the website or pretty much hold any logic that needs to run on the site.
+
+
+## Service Worker Analysis
+
+Out of the 10,640 PWAs, about 3940 of them had service workers that were detected by our site.
+ 
+ ![Alt text](/posts/data-insights-on-PWAs/appcap.png)
+
+What does this tell us? Well, for one thing, it shows that there is a lot of interest and potential in building PWAs, as evidenced by the large number of URLs that came to PWABuilder. However, it also reveals that there is a gap between the desire to create PWAs and the actual implementation of the core PWA features. Only about 37% of the PWAs had service workers, which means that  majority of them are missing out on the benefits of offline support, push notifications, background sync, and more. It is also a fair argument that not all PWAs need all of these features. Lately a lot of the major browsers have done away with mandating service workers for installability which could also mean developers see less value in building and maintaining service workers unless there is a business reason to use them.
```

Comment:
Should we link to resources on how to implement a service worker or somewhere to read up on service workers?

Reply:
```suggestion
What does this tell us? Well, for one thing, it shows that there is a lot of interest and potential in building PWAs, as evidenced by the large number of URLs that came to PWABuilder. However, it also reveals that there is a gap between the desire to create PWAs and the actual implementation of the core PWA features. Only about 37% of the PWAs had [service workers](https://docs.pwabuilder.com/#/home/sw-intro), which means that majority of them are missing out on the benefits of offline support, [push notifications](https://docs.pwabuilder.com/#/home/native-features?id=push-notifications), [background sync](https://docs.pwabuilder.com/#/home/native-features?id=background-sync), and more. It is also a fair argument that not all PWAs need all of these features. Lately a lot of the major browsers have done away with mandating service workers for installability which could also mean developers see less value in building and maintaining service workers unless there is a business reason to use them.
```